web-client/src/views/ColdCaseReport/ColdCaseReport.tsx

```diff
@@ -0,0 +1,36 @@
+import { BigHeader } from '../BigHeader';
+import { ColdCaseReportList } from './ColdCaseReportList';
+import { ErrorNotification } from '../ErrorNotification';
+import { SuccessNotification } from '../SuccessNotification';
+import { connect } from '@web-client/presenter/shared.cerebral';
+import { state } from '@web-client/presenter/app.cerebral';
+import React from 'react';
+
+export const ColdCaseReport = connect(
+  {
+    coldCaseReport: state.coldCaseReport,
+  },
+  function ColdCaseReport({ coldCaseReport }) {
+    return (
+      <>
+        <BigHeader text="Reports" />
+        <section className="usa-section grid-container">
+          <SuccessNotification />
+          <ErrorNotification />
+          <div className="title">
+            <h1>Cold Case Report</h1>
+          </div>
+
+          <div className="margin-bottom-2">
+            Showing cases not at issue with no activity in the last 120 days and
```

Comment:
Nit: Instead of hard-coding 120 days, it might be good to have this value be a property of the report that can be exposed to the components and interpolated in this text instead.

Reply:
How do you feel about the constant-based approach I just pushed?

Reply:
Perfect - thank you sir!